src/pages/HomePage/HomePage.style.ts

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+
+const HomePageContainer = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const HomePageWrapper = styled.div`
+  width: 128rem;
+`;
+
+const HomeSectionContainer = styled.main`
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: calc(50% - 1rem) calc(50% - 1rem);
+  grid-gap: 2rem;
+  height: 58rem;
+`;
```

Comment:
calc(50% - 1rem)을 해주신 이유가 궁금합니다!  👀

위의 코드를 바꿔서 그런지 카드가 겹치는 현상이 있어서 아래처럼 변경해주어야 할 것 같아요~

높이를 지정해주게 되면 grid의 1fr 속성에 의해서 빈공간이 생기게 돼서, height는 카드에서 지정해주는게 좋을 것 같습니다!

```suggestion
const HomeSectionContainer = styled.main`
  display: grid;
  grid-template-rows: 1fr 1fr;
  grid-template-columns: 1fr 1fr;
  gap: 2rem;
  width: 100%;
`;
```

Reply:
| calc(50% - 1rem)을 해주신 이유가 궁금합니다 👀

왼쪽의 50%는 총 2개의 행을 만들기 위함이고, 오른쪽의 1rem을 부여한 이유는
grid에 미숙하여, gap 2rem 속성을 부여하기 위해, 각 박스당 길이를 1rem씩 감하여 부여하였습니다.

말씀처럼 현재 코드에 height를 지정해주니 빈공간이 생기게 되어서 위와 같이 작성하였으나,
height를 카드에 지정해주니 잘 구현되었습니다! 👍